airbyte-integrations/connectors/source-asana/source_asana/manifest.yaml

```diff
@@ -0,0 +1,2622 @@
+      path: "/attachments"
+    retriever:
+      $ref: "#/definitions/retriever"
+      paginator:
+        $ref: "#/definitions/paginator_base"
+      requester:
+        $ref: "#/definitions/requester"
+        use_cache: true
+        request_parameters:
+          parent: "{{ stream_slice.parent_gid }}"
+      partition_router:
+        - type: SubstreamPartitionRouter
+          parent_stream_configs:
+            - stream: "#/definitions/projects_stream"
+              parent_key: "gid"
+              partition_field: "parent_gid"
+            - stream: "#/definitions/tasks_stream"
+              parent_key: "gid"
+              partition_field: "parent_gid"
+
+  attachments_compact_partition_router:
+    type: SubstreamPartitionRouter
+    parent_stream_configs:
+      - stream: "#/definitions/attachments_compact_stream"
+        parent_key: "gid"
+        partition_field: "attachments_gid"
+
+  attachments_stream:
+    $ref: "#/definitions/base_stream"
+    name: "attachments"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/attachments_schema"
+    $parameters:
+      path: "/attachments/{{ stream_slice['attachments_gid'] }}"
+    retriever:
+      $ref: "#/definitions/retriever"
+      paginator:
+        $ref: "#/definitions/paginator_base"
+      partition_router:
+        $ref: "#/definitions/attachments_compact_partition_router"
+
+  portfolios_compact_stream:
+    $ref: "#/definitions/base_stream"
+    name: "portfolios_compact"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/portfolios_compact_schema"
+    $parameters:
+      path: "/portfolios"
+    retriever:
+      $ref: "#/definitions/retriever"
+      requester:
+        $ref: "#/definitions/requester"
+        request_parameters:
+          owner: "{{ stream_slice['users_gid'] }}"
+          workspace: "{{ stream_slice['workspace_gid'] }}"
+      paginator:
+        $ref: "#/definitions/paginator_base"
+      partition_router:
+        - type: SubstreamPartitionRouter
+          parent_stream_configs:
+            - stream: "#/definitions/workspaces_stream"
+              parent_key: "gid"
+              partition_field: "workspace_gid"
+        - type: SubstreamPartitionRouter
+          parent_stream_configs:
+            - stream: "#/definitions/users_stream"
+              parent_key: "gid"
+              partition_field: "users_gid"
+
+  portfolios_compact_partition_router:
+    type: SubstreamPartitionRouter
+    parent_stream_configs:
+      - stream: "#/definitions/portfolios_compact_stream"
+        parent_key: "gid"
+        partition_field: "portfolios_gid"
+
+  portfolios_stream:
+    $ref: "#/definitions/base_stream"
+    name: "portfolios"
+    schema_loader:
+      type: InlineSchemaLoader
+      schema:
+        $ref: "#/definitions/portfolios_schema"
+    $parameters:
+      path: "/portfolios/{{ stream_slice['portfolios_gid'] }}"
+    retriever:
```

Comment:
Missing request parameter: `opt_fields`. Please add it according to the original implementation.